site/content/contribute/webapp/end-to-end-tests.md

```diff
@@ -14,12 +14,22 @@ Not familiar with Cypress? Here is some documentation that will help you get sta
 
 ## Quick Overview on How to Run E2E Testing
 
+### In Local Development Machine
 1.  Run a local server instance by initiating `make run` to the mattermost-server folder. Confirm that the instance has started successfully.
-   - Run `make test-data` to preload your server instance with initial seed data.  Generated data such as users are typically used for logging, etc.
-   - Each test case should handle the required system or account settings, but in case you encountered some unexpected error while testing, you may want to run the server with default config based on `mattermost-server/config/default.json`).
-2.  Change directory to `mattermost-webapp/e2e`, install npm dependencies by `npm i` and initiate `npm run cypress:run` in the command line. This will start full E2E testing. To run partial testing, you may initiate `npm run cypress:open` in the command line which will open its desktop app.  From there, you may select particular tests you would like to run.
+    - Run `make test-data` to preload your server instance with initial seed data.
+    - Each test case should handle the required system or user settings, but in case you encountered some unexpected error while testing, you may want to run the server with default config based on `mattermost-server/config/default.json`).
+2.  Change directory to `mattermost-webapp/e2e`, install npm dependencies by `npm i`.
+    - Initiate `npm run cypress:run` in the command line to start full E2E testing except for those specs at `/cypress/integration/enterprise` folder which requires Enterprise license to run successfully.
+    - Initiate `npm run cypress:open` in the command line to start partial testing depending on which spec is selected via Cypress' desktop app.
+    - Initiate via node CLI to selectively run specs based on test metadata, e.g. `node run_tests.js --group='@accessibility'` which will run all specs with `@accessibility` metadata.
 3.  Tests are executed according to your selection and will display whether the tests passed or failed.
 
+### In Continuous Integration Pipeline
+We are running all tests in our Continuous Integration (CI) pipeline. However, they are grouped according to test stability.
+1. __Daily production tests against development branch (master)__ - initiated on master branch by `node run_tests.js --stage='@prod'` in the command line. These are tests, known as production tests, which were selected and labeled with `@prod` [test metadata](/contribute/webapp/end-to-end-tests/#adding-test-metadata-on-spec-files). See <a target="_blank" href="https://community-release.mattermost.com/core/pl/i3kg97o1fir9pje7yi8wecd45r">link</a> for an example test run posted in our community channel.
+2. __Daily production tests against release branch__ - same as number 1 except the test is initiated against release branch. See <a target="_blank" href="https://community-release.mattermost.com/core/pl/mhtepy6p33gi9fp33fsek161qr">link</a> for an example test run.
```

Comment:
```suggestion
2. __Daily production tests against release branch__ - Same as above except the test is initiated against the release branch. See <a target="_blank" href="https://community-release.mattermost.com/core/pl/mhtepy6p33gi9fp33fsek161qr">link</a> for an example test run.
```

Reply:
I'm good per comment but I remember before that someone said to avoid positional argument like the word "above" when referring to a certain location of the article.